examples/sample-foundry-project/cannonfile.consumer.toml

```diff
@@ -3,18 +3,18 @@ version = "0.0.1"
 description = "Simple project to verify the functionality of cannon"
 keywords = ["sample", "consumer"]
 
-[setting.change_greeting2]
-defaultValue = "new greeting!"
+[var.change_greeting2]
+value = "new greeting!"
 
-[import.greeters]
+[pull.greeters]
 source = "greeter-foundry"
 
-[import.more_greeters]
+[provision.more_greeters]
```

Comment:
```suggestion
[deploy.more_greeters]
```

Reply:
I like this. The other consideration is (like the other point) differentiating the step name from how it's referenced in the context. Are you imagining 'contracts' stays the same?

Reply:
Yeah I recon contract is ok.
Tbh it does not matter that much how things  are called, what really does matter is that name matches when you declare and when you use a thing. 
If I call something contract.Greeter, I expect to reference it everywhere in the code as contract.Greeter and not anything else.

Reply:
Got it, so you'd just expect to reference with `contract` and `import` (singular)?

Reply:
Yes 100%